Move background information below training modules (#1440)
HSF/hsf.github.io: _training/curriculum.md

{% include list_of_selected_training_modules.html ids="uproot,grid,parallel,alpaka,yadagereana,doc,generators" %}

> ## <a name="bkginfo"></a>Background information and further reads

Reviewer: ```suggestion
## <a name="bkginfo"></a>Background information and further reading
```

Reviewer: Or even just "Additional information".

Author: ```suggestion
## <a name="bkginfo"></a>Additional information
```

> Training in software and computing are essential ingredients for the success of any HEP experiment. As most experiments have similar basic prerequisites (Unix shell, Python, C++, …) we want to join our efforts and create one introductory software training curriculum that serves HEP newcomers the software skills needed as they enter the field, and in parallel, instill best practices for writing software.

Reviewer: ```suggestion
Training in software and computing are essential ingredients for the success of any HEP experiment. As most experiments have similar basic prerequisites (Unix shell, Python, C++, …) we have joined efforts and created this introductory software training curriculum that teaches HEP newcomers the software skills they need as they enter the field, and in parallel, instills best practices for writing software.
```

> The curriculum is comprised of a set of *modules* (developed by HSF, Software Carpentries, and other organization), so that learners students can focus on what is most relevant to them.

Reviewer: ```suggestion
The curriculum is composed of a set of *modules* (developed by HSF, Software Carpentries, and other organizations), so that learners/students can focus on what is most relevant to them.
```
